inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/ConfigConstants.java

```diff
@@ -74,4 +74,6 @@ public class ConfigConstants {
     public static String HTTP = "http://";
     public static String HTTPS = "https://";
 
+    public static int MAX_MESSAGE_LENGTH = 500 * 1024;
```

Comment:
Can this package size be obtained through other means other than SDK settings, such as getting this value from the Manager?

Reply:
Now the transmission process of maxMsgSize configuration is as follows:
1. `DataProxy` is configured first
2. `maxMsgSize` is transmitted to the manager along with the `DataProxy` `heartbeat packet`（in the `extParams` field）
3. The `manager` stores the `extParams` information (containing the `maxMsgSize` field) in the database
4. `DataProxy` obtains `maxMsgSize` when calling the `/dataproxy/getIpList/{inlongGroupId}` interface to obtain proxy cluster information
5. `DataProxy` uses `maxMsgSize` to determine whether the data exceeds the length limit